src/main/HomeScreen.js

```diff
@@ -45,12 +45,15 @@ export default function HomeScreen(props: Props): Node {
       <OfflineNoticePlaceholder />
       <View style={styles.iconList}>
         <TopTabButtonGeneral
-          name="globe"
           onPress={() => {
             dispatch(doNarrow(HOME_NARROW));
           }}
         >
-          <Icon size={24} style={{ textAlign: 'center ' }} color={BRAND_COLOR} name="globe" />
+          <IconAllMessages
+            size={24}
+            style={{ textAlign: 'center', transform: [{ scale: -1 }] }}
```

Comment:
The `transform`/`scale` here is a neat trick.

It should be encapsulated inside `IconAllMessages`. That also makes a good place for a code comment with the information from this commit message:

> Web Zulip's all-messages icon is an inverted list icon, so we're
> swapping mobile's all-messages icon to match.

It's fine if that means `IconAllMessages` has a slightly longer function definition than the others in `Icons.js`.